GeosCore/photolysis_mod.F90

```diff
@@ -1603,15 +1611,19 @@ SUBROUTINE RD_PROF_NC( Input_Opt, State_Chm, RC )
 
     ! Test if the file exists and define an output string
     IF ( FileExists ) THEN
-       FileMsg = 'FAST-JX (RD_PROF_NC): Opening'
+       FileMsg = '(RD_PROF_NC): Opening'
     ELSE
-       FileMsg = 'FAST-JX (RD_PROF_NC): REQUIRED FILE NOT FOUND'
+       FileMsg = '(RD_PROF_NC): REQUIRED FILE NOT FOUND'
     ENDIF
 
     ! Write to stdout for both regular and dry-run simulations
     IF ( Input_Opt%amIRoot ) THEN
-       WRITE( 6, 300 ) TRIM( FileMsg ), TRIM( nc_path )
-300    FORMAT( a, ' ', a )
+#ifdef FASTJX
```

Comment:
Same comment as before. Do we need to add fast-jx and cloud-j if the full path is printed?